bin/smith-datadog

```diff
@@ -134,6 +144,7 @@ class Smith::Datadog
           opts.on("--host <s>", String, "Riemann host (default #{options[:host]})") { |v| options[:host] = v }
           opts.on("--port <i>", Integer, "Riemann port (default #{options[:port]})") { |v| options[:port] = v }
           opts.on("--tags <tag1,tag1,...>", Array, "Tags to add to the alert") { |t| options[:tags] = t }
+          opts.on("--exit-on-fail <s>", String, "Exit on fail") { |t| options[:exit_on_fail] = t }
```

Comment:
Should we use boolean?

[Trollop docs](http://manageiq.github.io/trollop/): All options are taken to be boolean flags, defaulting to false, unless you specify a default or a type.